### ISSUE TITLE
[Usable – M365 Admin Portal – Devices]: NVDA is not announcing position when focus is on 'Fully managed' & 'App managed' tab items in browse mode.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13198]
### Steps to reproduce:
Step 1: Open the given URL with valid credentials then 'Microsoft M365 admin' page will open.
Step 2: Navigate to 'Devices' menu item and expand it then activate “Active Devices” menu item.
Step 3: Turn on the NVDA, Navigate to “App Managed” & "Fully managed" tab items
Step 4: Verified whether NVDA is narrating the position of tab items or not.

### Actual behavior:
NVDA is not announcing position when focus is on 'Fully managed' & 'App managed' tab items in browse mode.

### Expected behavior:
NVDA should announce position when focus is on 'Fully managed' & 'App managed' tab items in browse mode.

### Additional Notes: 
1. Similar issue exists with menu items(Refresh, Remove company data, Selected & Change view) under 'App managed' tab item.
2. Similar issue exists with menu items(Refresh, Remove company data, Factory reset, Delete, Selected & Change view) under 'Fully managed' tab item.
3. Same issue exists with devices which are available under table as screen reader does not announce position of devices while navigating using arrow key.
4. Working fine with Narrator screen reader.

### System configuration
Application: M365 Admin Portal
URL: https://admin.microsoft.com/Adminportal/Home
Browser: Edge, Version 96.0.1054.62 (Official build) (64-bit)

#### NVDA installed/portable/running from source: 
Installed

#### NVDA version: 
NVDA Version 2021.1

#### Windows version: 
Window 11, Version 21H2 (OS Build 22000.376)

#### Does the issue still occur after restarting your computer? 
Yes

**Comment by josephsl** [comment]
Hi, can you try with NVDA 2021.3.1 and see if that makes any difference? Also, does this happen with latest Edge beta/dev/canary? Thanks.

**Comment by feerrenrut** [comment]
This needs further investigation by Microsoft developers. Please provide a minimal test case for us to investigate. I'm closing this issue for now.
